examples/webpack/main.js

```diff
@@ -0,0 +1,31 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/licenses/publicdomain/
+
+// Hello world example for webpack. See also
```

Comment:
Isn't there something missing in the second sentence, since it's seems to me that "See also" ought to be followed by something?